configuration.yaml

```diff
@@ -26,9 +26,9 @@ EPMC:
 GeneBurden:
   azPhewasBinary: gs://otar000-evidence_input/GeneBurden/data_files/azphewas-com-450k-phewas-binary
   azPhewasQuantitative: gs://otar000-evidence_input/GeneBurden/data_files/azphewas-com-450k-phewas-quantitative
-  azTraitMappings: gs://otar000-evidence_input/GeneBurden/data_files/AZ_Traits.xlsx
   curation: https://raw.githubusercontent.com/opentargets/curation/master/gene_burden/curated_evidence.tsv
-  inputBucket: gs://otar000-evidence_input/GeneBurden/data_files
+  genebass: gs://otar000-evidence_input/GeneBurden/data_files/genebass_entries
+  traitMappings: https://github.com/opentargets/curation/blob/master/mappings/disease/manual_string.tsv?raw=true
```

Comment:
Thanks! This is much better because in this way I can read the remote file with SparkFiles just fine. The general approach is something like:
```python
spark.sparkcontext.addFile(url)

spark.read.csv(SparkFiles.get('data.csv'))
```

Having the `raw` parameter at the tail of the url made the thing just cumbersome.